plot.py

```diff
@@ -625,6 +629,7 @@ def _check_default_field(self):
         
         This should only occur upon start up with a new file.
         '''
+        #XXX this need re-thinking
```

Comment:
Well I suppose this could move somewhere else. It is needed when opening a file type for the first time.